src/module-magespices-mage2dark/Plugin/Model/View/Design.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace MageSpices\Mage2Dark\Plugin\Model\View;
+
+use Magento\Framework\App\Area;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Stdlib\DateTime\TimezoneInterface;
+
+class Design
+{
+    protected $scopeConfig;
+
+    protected $timeZone;
+
+    protected $themeSelectionConfig;
+
+    public function __construct(
+        ScopeConfigInterface $scopeConfig,
+        TimezoneInterface $timezone
+    ) {
+        $this->scopeConfig = $scopeConfig;
+        $this->timeZone = $timezone;
+    }
+
+    /**
+     * @param \Magento\Theme\Model\View\Design $subject
+     * @param callable $proceed
+     * @param string|null $area
+     * @param array $params
+     * @return
+     */
+    public function aroundGetConfigurationDesignTheme(
```

Comment:
I believe it's safer to use `after` plugin and override the result than use around :-)

Reply:
Actually, I need area param to apply customization only if it is admin area. That's why I used around plugin instead of after.

But as the plugin actually has only admin scope(added in di.xml under adminhtml) so, I believe it is safe to use after plugin without worrying about area param. will modify and test it.

Reply:
This is very common mistake.
After plugin has access to method parameter too! :-) check it out. https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html

Reply:
OMG! 😮 I didn't know it. Will alter around implementation to after.

Thanks for sharing this!

Reply:
Done ✅